Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector.py

```diff
@@ -270,18 +271,24 @@ def main() -> None:  # pragma: no cover
             else:
                 events = integration_context.get('events')
                 demisto.info('fetching events from integration context')
-            try:
-                demisto.info(f'sending the following amount of events into XSIAM: {len(events)}')
-                send_events_to_xsiam(
-                    events=events,
-                    vendor=VENDOR,
-                    product=PRODUCT
-                )
-                demisto.setIntegrationContext({})
-            except Exception as e:
-                demisto.info(f'got error when trying to send events to XSIAM: [{e}]')
-                demisto.setIntegrationContext({'events': events})
-                demisto.info(f'set the following events into integration context: {events}')
+            for i in range(1, 4):
+                try:
+                    demisto.info(f'sending the following amount of events into XSIAM: {len(events)}, retry number {i}')
+                    send_events_to_xsiam(
```

Comment:
I think we better insert this fix to the send_events_to_xsiam function in CommonServerPython script.

Reply:
no problem, will be done.